app/pages/Profile/agriculture/CropsAreaVsValue.jsx

```diff
@@ -25,9 +25,9 @@ class CropsAreaVsValue extends SectionColumns {
     const bottomCrop = crops[crops.length - 1];
 
     return (
-      <SectionColumns>
-        <SectionTitle>Harvested Area Versus Value of Production</SectionTitle>
-        <article>
+      <SectionRows>
+        <SectionTitle className="sectionTitle">Harvested Area Versus Value of Production</SectionTitle>
```

Comment:
Yeah it works, although I'm realizing your element <SectionTitle> becomes an <h4> when viewing in the browser but it still has the class "sectionTitle". Is that something that happens when the site compiles?